### DISCUSSION TITLE
user account is blocked

### DISCUSSION THREAD

**Discussion by y52047** [vercel/community#3927] [Account Issue] [answered]
### Question

I want to create blog with vercel. But the user account is blocked.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
